src/mudlet-lua/lua/geyser/GeyserLabel.lua

```diff
@@ -95,6 +99,19 @@ function Geyser.Label:processFormatString(format)
   end
 end
 
+--- Sets the font face for the label, use empty string to clear the font and use css/default. Returns true if the font changed, nil+error if not.
+-- @param font font face to use
+function Geyser.Label:setFont(font)
+  local af = getAvailableFonts()
+  if not (af[font] or font == "") then
+    local err = "attempt to call setFont with font '" .. font .. "' which is not available, see getAvailableFonts() for valid options\n"
```

Comment:
```suggestion
    local err = "Geyser.Label:setFont(): attempt to call setFont with font '" .. font .. "' which is not available, see getAvailableFonts() for valid options\n"
```
It's hard to see where the error msg is coming from otherwise, no context:

![image](https://user-images.githubusercontent.com/110988/75805600-6b90ba00-5d82-11ea-82c6-dd63813034b3.png)